### DISCUSSION TITLE
Obtener mensajes de error o exito

### DISCUSSION THREAD

**Discussion by DavidKingV** [netflie/whatsapp-cloud-api#44] [Q&A]
Hola, he logrado enviar mensajes sin problema, solo quisiera saber si hay alguna manera de obtener una respuesta de error en caso de que el mensaje no se envié. 
Ya que uso sweetalert2 para que salga algún mensaje de éxito o error dependiendo de la respuesta del la solicitud.

**Comment by aalbarca** [comment]
Hola @DavidKingV 

Cuando mandas un mensaje recibes como respuesta una instancia de la clase [Response](https://github.com/netflie/whatsapp-cloud-api/blob/main/src/Response.php). Revisa la API de esta clase y encontrarás algunos métodos que quizás te sean útiles. Por ejemplo, puedes utilizar el método `isError()` para saber si se ha producido un error o no. También los métodos `decodedBody` y `body` para ver la respuesta devuelva por Facebook (aka Meta).

¡Saludos!

**Reply by DavidKingV** [reply]
Me podrías dar un pequeño ejemplo? estoy un poco perdido

**Reply by aboshhab12** [reply]
> Hello @DavidKingV
> 
> When you send a message, you receive an instance of the [Response](https://github.com/netflie/whatsapp-cloud-api/blob/main/src/Response.php) class in response. Check out the API for this class and you'll find some methods that you might find useful. For example, you can use the method `isError()` to find out if an error has occurred or not. Also the methods `decodedBody` and `body` to see the answer return by Facebook (aka Meta).
> 
> Best regards!

is there an example how to use it?
